music_assistant/server/providers/deezer/__init__.py

```diff
@@ -530,7 +528,9 @@ def parse_track(
         extra_init_kwargs: dict[str, Any] | None = None,
     ) -> Track | PlaylistTrack:
         """Parse the deezer-python track to a MASS track."""
-        if "position" in extra_init_kwargs:
+        if extra_init_kwargs is None:
+            extra_init_kwargs = {}
+        elif "position" in extra_init_kwargs:
```

Comment:
Like this?

Reply:
yes and then adjust line 569 and remove the 'or {}'

Reply:
Wait that cant be right since track_class isnt asigned if extra_init_kwargs is None

Reply:
set it in the first if then